tutorials/tutorial.ipynb

```diff
@@ -0,0 +1,1492 @@
+{
```

Comment:
<p>Perhaps just a bit more interpretation here. What would this mean?</p><p>For example "This means that on average XX is coordinated to XX XX% of the time"</p><p> </p><p>Also the CNs for FEC and PF6  seem very low, this is a cross CN with Li right?</p><p> </p><p>Naively I might  this to mean that FEC has a molecule in its first shell 0.25% of the time, which doesn't seem right. If it is a cross Li-PF6 CN then we should be as explicit as possible.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MDAnalysis/solvation-analysis/pull/36/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>

Reply:
<p>Good point, I added more detail as suggested.</p><p> </p><p>In this system, BN coordinates extremely strongly with Li and largely excludes FEC and PF6, so the CNs are quite low.</p>

Reply:
I would also add a note about this in the docs for `Coordination`, that it must be interpreted as a species-species CN.